mlptrain/sampling/md.py

```diff
@@ -64,6 +67,13 @@ def run_mlp_md(
 
         interval: (int) Interval between saving the geometry
 
+        pressure: pressure in bar to run Berendsen NPT MD, temperature
```

Comment:
nPT is currently not supported in active learning, it might be worth to add a note here

Reply:
Good point, I will make sure to explain this here